src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -690,7 +691,36 @@ public void checkSpotlights() {
 			WebElement slot = waitForElementByCss(spotlightSelector + " img");
 			verifySlotExpanded(slot);
 
-			adsComparison.isAdVisible(slot, spotlightSelector, driver);
+			adsComparison.isAdVisible(slot, spotlightSelector, driver, false);
 		}
 	}
+
+	public AdsBaseObject verifySize(String slotName, int slotWidth, int slotHeight) {
+		WebElement element = getWebElement(slotName);
+		Dimension size = element.getSize();
+		Assertion.assertEquals(size.getWidth(), slotWidth);
+		Assertion.assertEquals(size.getHeight(), slotHeight);
+		PageObjectLogging.log("verifySize", slotName + " has size " + size, true, driver);
+		return this;
+	}
+
+	public AdsBaseObject verifyLineItemId(String slotName, String src, int lineItemId) {
+		String lineItemParam = getGptParams(slotName, src, GPT_DATA_ATTRIBUTES[0]);
+		Assertion.assertStringContains(String.valueOf(lineItemId), lineItemParam);
+		PageObjectLogging.log("verifyLineItemId", slotName + " has following line item: " + lineItemParam, true);
+		return this;
+	}
+
+	public AdsBaseObject verifyAdImage(String slotName, String src, String imageUrl) {
+		WebElement element = getWebElement(slotName);
+		boolean isMobile = src.toUpperCase().equals("MOBILE");
```

Comment:
swap the left and right for the equals to avoid NullPointerException
also, use equalsIgnoreCase instead
```
"MOBILE".equalsIgnoreCase(src);
```